docs/detections/rules-ui-manage.asciidoc

```diff
@@ -135,9 +135,11 @@ Similarly, rules will be skipped if they can't be modified by a bulk edit. For e
 * Bulk edit multiple rules: Select the rules you want to edit, then select an action from the *Bulk actions* menu:
 ** *Index patterns*: Add or delete the index patterns used by all selected rules.
 ** *Tags*: Add or delete tags on all selected rules.
-** *Add rule actions*: Add <<rule-notifications,notification actions>> on all selected rules.
+** *Add rule actions*: Add <<rule-notifications,notification actions>> on all selected rules. The action frequency you select applies to all actions (both new and existing) on all selected rules. If you don't want to change the frequency of existing actions, update the rules separately.
+
 +
-NOTE: The action frequency you select applies to all actions (both new and existing) on all selected rules. If you don't want to change the frequency of existing actions, update the rules separately.
+IMPORTANT: After upgrading to 8.8, frequency settings for rule actions created in 8.7 or earlier are moved from the rule level to the action level. The action schedules remain the same and will continue to run on their previously specified frequency (`On each rule execution` `Hourly`, `Daily`, or `Weekly`). Rules without actions will be snoozed indefinitely. 
```

Comment:
I think we should remove last sentence about snoozing.

There are several cases there: 1. when user created a rule without actions it won't be snoozed, it will just not do notifications because of the lack of the actions; 2. when user created a rule with some actions and then updated the rule level frequency to "Perform no actions", in this case we will snooze the rule.

I think it might be confusing for the user to see those cases. they will notice it via updated UI where we will show that rule has been snoozed.

What do you think @peluja1012

Reply:
@e40pud would you mind taking a look at my testing notes [here](https://docs.google.com/document/d/16R2Q7_h4VGTlgduzXoMtkrEnScCr4f92avok6NJgSvQ/edit)? I'm a little confused because I found that rules with actions and the "Perform no actions" frequency selected _do not_ get snoozed when you import it into 8.8. 

If I found a bug and that rule _should_ be snoozed in 8.8, would the following work:
> Rules with actions that were turned off in 8.7 (the `Perform no actions` action frequency was selected) are snoozed indefinitely after the upgrade.

Reply:
@e40pud @nastasha-solomon I've added a [migration](https://github.com/elastic/kibana/pull/156593) to unmute rules upon `8.7` -> `8.8` migration. `muteAll` has become a part of snooze settings so rules won't be snoozed by default after the migration.